README.md

```diff
@@ -50,7 +50,7 @@ This project provides:
 
 ## API
 
-<h4 name="TypeClass"><code><a href="https://github.com/sanctuary-js/sanctuary-type-classes/blob/v1.0.0/index.js#L105">TypeClass :: (String, Array TypeClass, a -> Boolean) -> TypeClass</a></code></h4>
+<h4 name="TypeClass"><code><a href="https://github.com/sanctuary-js/sanctuary-type-classes/blob/v1.0.0/index.js#L112">TypeClass :: (String, Array TypeClass, a -> Boolean) -> TypeClass</a></code></h4>
```

Comment:
As mentioned in the [__CONTRIBUTING.md__][1], it's not necessary to update __README.md__ manually. ;)


[1]: https://github.com/sanctuary-js/sanctuary-type-classes/blob/master/CONTRIBUTING.md